internal/provider/resource_service_user_token_test.go

```diff
@@ -14,7 +14,7 @@ func TestAccResourceServiceUserToken(t *testing.T) {
 		tokenId     = "test-token-id"
 		description = "Test token description"
 	)
-	expiresAt := time.Now().Add(24 * time.Hour).Format("2006-01-02T15:04:05.999Z")
+	expiresAt := time.Now().Add(24 * time.Hour).Format("2006-01-02T15:04:05Z")
```

Comment:
Nit: You can also use `time.RFC3339` for the format, which should result in the same :-)